oct/tests/__init__.py

```diff
@@ -1,26 +1,31 @@
+import logging
+import os
+from oct.device_connection import NewSession
```

Comment:
this is an internal import and should come after all built-in and installed package imports

Reply:
I know, I saw it in Checks.
Created this pr just to see what Dima will say about log logic